src/pages/RFOX/hooks/useRfoxContext.tsx

```diff
@@ -32,9 +32,7 @@ export const RFOXProvider: React.FC<React.PropsWithChildren<{ stakingAssetId: As
 
   const filter = useMemo(
     () =>
-      stakingAssetAccountId && stakingAssetId
-        ? { assetId: stakingAssetAccountId, accountId: stakingAssetAccountId }
-        : undefined,
+      stakingAssetAccountId && stakingAssetId ? { accountId: stakingAssetAccountId } : undefined,
```

Comment:
@NeOMakinG mergefix issues?

Reply:
Yep fixed in [2b38bd6](https://github.com/shapeshift/web/pull/7952/commits/2b38bd691a2b53a6c5fc056cd4fe189082ca3806)